UniAtHome/UniAtHome.DAL/Repositories/UsersRepository.cs

```diff
@@ -1,17 +1,29 @@
 ﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
 using UniAtHome.DAL.Entities;
 
 namespace UniAtHome.DAL.Repositories
 {
     public sealed class UsersRepository
     {
-        private UserManager<User> userManager;
+        private readonly UniAtHomeDbContext context;
 
-        public UsersRepository(UserManager<User> userManager)
+        private readonly UserManager<User> userManager;
+
+        private readonly int refreshTokenLifetime;
+
+        public UsersRepository(
+            UniAtHomeDbContext context,
```

Comment:
Don't use UniAtHomeDbContext. We should use DbContext and then make Set<T>. (IoC)